controllers/apply/form-router/lib/salesforce.js

```diff
@@ -1,78 +1,69 @@
 'use strict';
-const request = require('request-promise-native');
+const got = require('got');
+const FormData = require('form-data');
 const { SALESFORCE_AUTH } = require('../../../../common/secrets');
 
 class Salesforce {
     constructor(apiUrl, accessToken) {
-        this.apiUrl = apiUrl;
         this.apiVersion = 'v45.0';
-        this.headers = { Authorization: `Bearer ${accessToken}` };
-    }
-    get(urlPath) {
```

Comment:
Got rid of this get helper as a) we don't use it and b) even if we did it's no longer any more code than using the got instance directly.